index.html

```diff
@@ -1,285 +1,357 @@
+                      alt="twitter"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img
+                      class="socialIcon"
+                      src="./assets/socialIcon.png"
+                      alt="social"
+                  /></a>
+                </li>
+              </ul>
+            </div>
+            <button type="button">Get my resume</button>
+          </div>
         </div>
+        <div class="languages-framework-skills">
+          <ul class="list-title">
+            <li>
+              <h2>Languages</h2>
+              <img class="arrowdown" src="./assets/arrowright.png" alt="" />
+            </li>
+            <ul class="languages-list-items-container">
+              <li class="languages-list-items">
+                <div>
+                  <img src="./assets/Javascript_ellipse 1.png" alt="" />
+                  <h2>Javascript</h2>
+                </div>
+              </li>
+              <li class="languages-list-items">
+                <div>
+                    <img src="./assets/html_ellipse 1.png" alt="" />
+                  <h2>HTML</h2>
+                </div>
+              </li>
+              <li class="languages-list-items">
+                <div>
+                  <img src="./assets/cssecllipse.png" alt="" />
+                  <h2>CSS</h2>
+                </div>
+              </li>
+            </ul>
+          </ul>
+          
+          <ul class="list-title">
+            <li>
+              <h2>Frameworks</h2>
+               <img class="arrowright" src="./assets/arrowright.png" alt="" />
+            </li>
+          </ul>
+            <hr class="separator">
+          <ul class="list-title">
+            <li>
+              <h2>Skills.</h2>
+              <img class="arrowright" src="./assets/arrowright.png" alt="" />
+            </li>
+          </ul>
+           <hr class="separator"/>
+    
+        </div>
+      </div>
     </section>
 
-
-
-
-
-
-
-</body>
+    <section id="form-container">
+      <div>
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+      <form 
+        action="https://formspree.io/f/mdovrprr"
+        method="POST">
+        <ul>
+          <li>
+            <input type="text" maxlength="30"  placeholder="Lorem Ipsum" required />
+          </li>
+          <li>
+            <input type="email" placeholder="loremipsum@gmail.com" required />
+          </li>
+          <li>
+            <textarea
+              name="message"
+              cols="28"
+              rows="7"
+              maxlength="500"
+              required
+              placeholder="Write you message here..."
+            ></textarea>
+          </li>
+          <li>
+            <button type="button">Get in touch</button>
```

Comment:
Your form is currently not submitting any data and that is because your button has a type of `button`. Please update this button to have a type of `submit` so that your form makes successful submissions🙏